packages/visualizations/src/components/Table/Cell/Cell.svelte

```diff
@@ -21,6 +22,10 @@
         padding: var(--spacing-75);
     }
 
+    :global(.ods-dataviz--default td.rtl-direction) {
+        text-align: right;
+    }
```

Comment:
Using `css` to set direction seems not recommended

https://www.w3.org/International/questions/qa-html-dir.en.html#no_css